doc/content/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Home"
+description: ""
+weight: 1
+draft: false
```

Comment:
I think `draft` is `false` by default, so then we can just leave it out, because we won't have any drafts.